src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -704,6 +704,15 @@ void Unit::DealDamageMods(Unit const* victim, uint32 &damage, uint32* absorb) co
 
 uint32 Unit::DealDamage(Unit* victim, uint32 damage, CleanDamage const* cleanDamage, DamageEffectType damagetype, SpellSchoolMask damageSchoolMask, SpellInfo const* spellProto, bool durabilityLoss)
 {
+    // Sparring Checks
+    if (Creature* me = ToCreature())
+        if (Creature* target = victim->ToCreature())
+            if (me->CanSparrWith(target))
+                if (damage >= victim->GetHealth) // safety check to prevent creatures getting accidentally killed when the percentage is small
```

Comment:
```
src/server/game/Entities/Unit/Unit.cpp:711:39: fatal error:
 reference to non-static member function must be called;
 did you mean to call it with no arguments?
                if (damage >= victim->GetHealth) // safety check to prevent creatures getting accidentally killed when the percentage is small
                              ~~~~~~~~^~~~~~~~~
                                               ()
```

Reply:
I really dont know what is happening there.... looks like missing header includes but I dont wanna pollute the includes again

Reply:
GetHealth  -> GetHealth()